src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -36,7 +36,9 @@ class OutgoingMigration::SliceSlotMigration : private ProtocolClient {
   SliceSlotMigration(DbSlice* slice, ServerContext server_context, SlotSet slots,
                      journal::Journal* journal, OutgoingMigration* om)
       : ProtocolClient(server_context), streamer_(slice, std::move(slots), journal, &cntx_) {
-    cntx_.SwitchErrorHandler([om](auto ge) { om->Finish(std::move(ge)); });
+    cntx_.SwitchErrorHandler([om](auto ge) {
+      om->Finish(ge != errc::operation_canceled ? std::move(ge) : GenericError());
```

Comment:
Why are we purging `operation_cancelled` and replacing it with an `empty` `GenericError` ?